core/server/apps/amp/lib/helpers/amp_content.js

```diff
@@ -121,10 +122,17 @@ function getAmperizeHTML(html, post) {
     html = makeAbsoluteUrl(html, config.url, post.url).html();
 
     if (!amperizeCache[post.id] || moment(new Date(amperizeCache[post.id].updated_at)).diff(new Date(post.updated_at)) < 0) {
-        return new Promise(function (resolve, reject) {
+        return new Promise(function (resolve) {
+
             amperize.parse(html, function (err, res) {
                 if (err) {
-                    return reject(err);
+                    errors.logError(err.message, err.src);
```

Comment:
src is undefined for me?

maybe we can match two types of error:
1. is a timeout for images and the error context is something like "Your image couldn't get fetched"
2. other errors and we only print a general context like "Amp could not get parsed"